backend/src/main/java/com/bakdata/conquery/mode/local/LocalManagerProvider.java

```diff
@@ -54,4 +51,11 @@ public DelegateManager<LocalNamespace> provideManager(ConqueryConfig config, Env
 				ManagerProvider.newJobManager(config)
 		);
 	}
+
+	protected SqlDialect createSqlDialect(SqlConnectorConfig sqlConnectorConfig, DSLContext dslContext) {
```

Comment:
Dadurch kann ich die Methode in einer Testklasse einfach mit den entsprechenden Test-Dialekten überschreiben.